warehouse/forklift/legacy.py

```diff
@@ -897,13 +897,13 @@ def file_upload(request):
                     if request.oidc_publisher
                     else None
                 ),
-                "reusable_worfklow_used": (
-                    is_from_reusable_workflow(
-                        request.oidc_publisher, request.oidc_claims
-                    )
-                    if request.oidc_publisher
-                    else False
-                ),
+                # NOTE(ww, 2024-10-21): This unfortunately typo'd event field
+                # was always incorrect, since the `file_upload` endpoint isn't
+                # itself aware of the kind of workflow that minted its API
+                # token. The correctly named datapoint is now under the
+                # `account:short_lived_api_token:added` event which, despite
+                # its name, is actually a project event.
+                "reusable_worfklow_used": False,
```

Comment:
NB: I _think_ we could also just remove this outright: there should be no dependencies on this event field at all, since we added it solely to collect figures on reusable workflow use.

Reply:
I'd prefer removal if we considered it transient only, less context to carry forward.

Reply:
Makes sense, removing!